### PR TITLE
feat(benchmark-wordpress): Add data update script

### PR DESCRIPTION
**Pull request by TylerBarnes**
This was copied from the Drupal benchmark and modified to update WPGraphQL instead.

### CODE REVIEW

File: benchmarks/source-wordpress/package.json [hunk 1]
```diff
@@ -23,12 +23,13 @@
     "gatsby-transformer-sharp": "^2.3.14",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "ts-node": "^8.9.0",
     "typescript": "^3.8.3"
```
Comment (duffn) [inline benchmarks/source-wordpress/package.json:26]: Don't need this anymore.

Comment (TylerBarnes) [reply benchmarks/source-wordpress/package.json]: @duffn this diff is it being removed :p
This did prompt me to remove 2 other unused packages tho!

Comment (duffn) [reply benchmarks/source-wordpress/package.json]: Oops, did I tag the wrong line? I was actually trying to mention `typescript` but I'm glad it prompted that anyway!

Comment (TylerBarnes) [reply benchmarks/source-wordpress/package.json]: Oh 🤦 that makes more sense hahah. Thanks Duffy!

**Pull request merged**